<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FTIR Data Acquisition Software</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icons/spectrometer.png</normaloff>icons/spectrometer.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="PlotWidget" name="GraphWidget" native="true">
      <widget class="PlotWidget" name="widget" native="true">
       <property name="geometry">
        <rect>
         <x>9</x>
         <y>9</y>
         <width>771</width>
         <height>481</height>
        </rect>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuSerial">
    <property name="title">
     <string>Serial</string>
    </property>
    <addaction name="actionConnect"/>
    <addaction name="actionDisconnect"/>
    <addaction name="action_Capture"/>
    <addaction name="actionLiveView"/>
   </widget>
   <widget class="QMenu" name="menuConfig">
    <property name="title">
     <string>&amp;Settings</string>
    </property>
    <addaction name="actionConfig"/>
   </widget>
   <addaction name="menuMenu"/>
   <addaction name="menuSerial"/>
   <addaction name="menuConfig"/>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="statusTip">
    <string extracomment="Status here..."/>
   </property>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
   <addaction name="separator"/>
   <addaction name="actionConnect"/>
   <addaction name="actionDisconnect"/>
   <addaction name="actionLiveView"/>
   <addaction name="action_Capture"/>
   <addaction name="separator"/>
   <addaction name="actionConfig"/>
   <addaction name="separator"/>
   <addaction name="actionHelp"/>
   <addaction name="separator"/>
  </widget>
  <action name="actionOpen">
   <property name="icon">
    <iconset>
     <normaloff>icons/open.png</normaloff>icons/open.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Open</string>
   </property>
   <property name="toolTip">
    <string>Open existing file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset>
     <normaloff>icons/floppy-disk.png</normaloff>icons/floppy-disk.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Save</string>
   </property>
   <property name="toolTip">
    <string>Save the data</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset>
     <normaloff>icons/logout.png</normaloff>icons/logout.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Exit</string>
   </property>
   <property name="toolTip">
    <string>Exit the application</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="actionConnect">
   <property name="icon">
    <iconset>
     <normaloff>icons/link.png</normaloff>icons/link.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Connect</string>
   </property>
   <property name="toolTip">
    <string>Connect through serial port</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionDisconnect">
   <property name="icon">
    <iconset>
     <normaloff>icons/broken-link.png</normaloff>icons/broken-link.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Disconnect</string>
   </property>
   <property name="toolTip">
    <string>Disconnect serial port</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="actionConfig">
   <property name="icon">
    <iconset>
     <normaloff>icons/settings.png</normaloff>icons/settings.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Config</string>
   </property>
   <property name="toolTip">
    <string>Configeration for settings</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+K</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="icon">
    <iconset>
     <normaloff>icons/information.png</normaloff>icons/information.png</iconset>
   </property>
   <property name="text">
    <string>Help</string>
   </property>
   <property name="toolTip">
    <string>About </string>
   </property>
  </action>
  <action name="action_Capture">
   <property name="icon">
    <iconset>
     <normaloff>icons/audio-waves.png</normaloff>icons/audio-waves.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Capture</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+M</string>
   </property>
  </action>
  <action name="actionLiveView">
   <property name="icon">
    <iconset>
     <normaloff>icons/live-news.png</normaloff>icons/live-news.png</iconset>
   </property>
   <property name="text">
    <string>LiveView</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QWidget</extends>
   <header location="global">plotwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
